pkg/disttask/framework/scheduler/collector.go

```diff
@@ -37,13 +37,20 @@ func init() {
 // Therefore, a custom collector is used.
 type collector struct {
 	subtaskInfo atomic.Pointer[[]*proto.SubtaskBase]
+	taskInfo    atomic.Pointer[[]*proto.TaskBase]
 
+	tasks           *prometheus.Desc
 	subtasks        *prometheus.Desc
 	subtaskDuration *prometheus.Desc
 }
 
 func newCollector() *collector {
 	return &collector{
+		tasks: prometheus.NewDesc(
+			"tidb_disttask_task_status",
```

Comment:
```suggestion
			"tidb_disttask_tasks",
```
we have the `status` in the label